websites/Z/Zybooks/metadata.json

```diff
@@ -0,0 +1,21 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.9",
+	"author": {
+		"id": "199605025914224641",
+		"name": "pokestarfan"
+	},
+	"service": "Zybooks",
+	"description": {
+		"en": "Share progress on Zybooks activities."
+	},
+	"url": "learn.zybooks.com",
+	"version": "1.0.0",
+	"logo": "https://i.imgur.com/viwt0Wj.png",
+	"thumbnail": "https://i.imgur.com/I4RTS73.png",
```

Comment:
The thumbnail may be a bit too small/low-res. A better one could is a screenshot of the website.

Reply:
Ensure that this is resolved